packages/in_app_purchase_tizen/tizen/src/in_app_purchase_tizen_plugin.cc

```diff
@@ -0,0 +1,57 @@
+#include "in_app_purchase_tizen_plugin.h"
+
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_method_codec.h>
+
+#include <memory>
+#include <string>
+
+#include "billing_manager.h"
+#include "log.h"
+
+class InAppPurchaseTizenPlugin : public flutter::Plugin {
+ public:
+  static void RegisterWithRegistrar(
+      FlutterDesktopPluginRegistrarRef registrar_ref,
+      flutter::PluginRegistrar *plugin_registrar);
+  InAppPurchaseTizenPlugin(FlutterDesktopPluginRegistrarRef registrar_ref,
```

Comment:
we can remove registrar_ref, because just need plugin_registrar when create method channel.

Reply:
fixed